decision-making/voting-on-proposals.md

```diff
@@ -0,0 +1,30 @@
+# Decision-making
+
+### Proposal submission
+Proposal is submitted by opening pull request to this repository.
+Initiator must present the idea on the architecture meeting.
+The announcement should include problem description the proposal is trying to resolve as well as solution(s).
+
+### Discussion
+There must be minimum 2 weeks of discussion in the pull request thread.
+After that initiator may decide when to start the vote.
+
+### Voting
+All eligible voters must get notification about the voting start.
+A valid voting period must be announced when voting is started, must be greater that 2 weeks and must not be changed during the vote.
+
+### Approval
+The number of `approve` votes must be greater than or equal to the number of `reject` votes multiplied by two in order to merge pull request.
```

Comment:
Maybe it's just me but wording seems complicated. Should we just stick to 2/3?

Reply:
So the "shall we introduce this" vote needs 2/3 and possible "how shall we introduce this" need majority. so with 3 choices 1/3+1 might be sufficient..